grpcdogfood/server/dogfood_server.go

```diff
@@ -16,26 +16,26 @@ import (
 	emptypb "google.golang.org/protobuf/types/known/emptypb"
 )
 
-const serverAddr = "localhost:50051"
+const port = ":50051"
 
 type chaosDogfoodServer struct {
 	pb.UnimplementedChaosDogfoodServer
 }
 
 func (s *chaosDogfoodServer) Order(ctx context.Context, req *pb.FoodRequest) (*pb.FoodReply, error) {
-	fmt.Printf("* %v food ordered\n", req.Animal)
+	fmt.Printf("| %v food ordered\n", req.Animal)
 	return &pb.FoodReply{Message: "Mock Reply", ConfirmationId: 1}, nil
 }
 
 func (s *chaosDogfoodServer) GetCatalog(ctx context.Context, req *emptypb.Empty) (*pb.CatalogReply, error) {
-	fmt.Println("* catalog delivered")
+	fmt.Println("x\n| catalog delivered")
 	return &pb.CatalogReply{Items: []*pb.CatalogItem{}}, nil
 }
 
 func main() {
-	fmt.Printf("listening on %v...\n", serverAddr)
+	fmt.Printf("listening on port%v...\n", port)
```

Comment:
`localhost:50051` does not work from the container. I have not been able to pin down the fix. Currently, the lint CI job complains that it's insecure to target all interfaces. Realistically, there's no more security issue from applying to "all" interfaces of a container inside a local Minikube cluster.

Will keep seeking a resolution, but perhaps we can just have this line bypass linting?

Reply:
I am looking into using the container's IP address instead of localhost